x/operator/client/cli/tx.go

```diff
@@ -73,10 +74,12 @@ func CmdRegisterOperator() *cobra.Command {
 		},
 	}
 
-	// EarningsAddr is the same as the sender's address, since the operator registration must be
-	// done by the operators themselves.
-
 	f := cmd.Flags()
+	// EarningAddr may be different from the sender's address.
+	f.String(
+		FlagEarningAddr, "", "The address which is used to receive the earning reward in the Exocore chain. "+
+			"If not provided, it will default to the sender's address.",
+	)
 	// ApproveAddr may be different from the sender's address.
 	f.String(
 		FlagApproveAddr, "", "The address which is used to approve the delegations made to "+
```

Comment:
Is this used anywhere?

Reply:
I removed the check that enforced the earning address to be the same as the operator address when setting operator states. So added a flag to allow operators to set a different earning address during registration. However, since the earning address isn't currently being used, I’m unsure if we should allow it to be different from the operator address.